woocommerce/woocommerce
# CYS: Reset products when the AI is offline

> [!IMPORTANT]  
> You need to test this PR using a build of WooCommerce Blocks based on https://github.com/woocommerce/woocommerce-blocks/pull/12076

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #41921 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a new site either via local environment or JN. Make sure to install WooCommerce Beta Tester as well.
2. Connect your site with Jetpack
3. Enable `customize-store` feature flag in `Tools -> WCA Test Helper -> Features`
4. Install and activate  the nightly build of WooCommerce Blocks [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce/files/13589648/woocommerce-gutenberg-products-block.zip).
5. Go to `WooCommerce -> Home` and click on `Customize your store` task.
6. Insert a prompt and follow the setup flow.
7. Ensure that the products match the prompt.


#### AI is offline

1.  Go to `WooCommerce -> Home` and click on `Customize your store` task.
2. Open your browser's inspector and use Chrome's content override [feature](https://developer.chrome.com/docs/devtools/overrides/) to change the content of https://status.openai.com/api/v2/status.json call. Change `status.indicator` to `major`. 
3. Perform a hard refresh
4. Click on `Design with AI` button and proceed to the assembler hub.
5.  Insert a prompt and follow the setup flow.
6.  Ensure that the products match the image below:


![image](https://github.com/woocommerce/woocommerce/assets/4463174/51b2faa6-cc8c-41fe-9ce7-7c21f8ed9ac5)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS: Reset products when the AI is offline.

#### Comment 

</details>
